### ISSUE TITLE
Why is package name not expfactoryr?

### ISSUE THREAD

**Issue by vsoch** [expfactory/expfactoryr#7]
hey @earcanal !

I'm testing a build, and I'd expect to be able to do:

```bash
R -e "library(expfactoryr); sessionInfo()"
```
but that doesn't work! Instead I have to do:

```bash
R -e "library(expfactory); sessionInfo()"
```
Is this intentional / the standard for most packages? if so, how do I reliably get the package name to load?

**Comment by earcanal** [comment]
TL;DR the R package is called `expfactory`.

I think the dissonance is because it seemed better from an R pov to call it `expfactory` as it's obviously an R package. From a github pov there was already the `expfactory` repo, so I think we called it `expfactoryr` to indicate what it is!

I still find [the advice about naming R packages/namespaces ](http://r-pkgs.had.co.nz/package.html) a bit clunky. Do you remember we eventually decided on `expfactory.$task` for task-specific packages? This is another good reason to have `expfactory` as the package for generic code.

I think there's an issue somewhere where we discussed all of this?

**Comment by vsoch** [comment]
Yes I remember! The name is fine, I was just looking for a programmatic way to find it (and I found it in the DESCRIPTION file) so we are good! Thanks :)
